inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ set(LINK_LIBRARIES
     inference_engine_snippets
     offline_transformations
     inference_engine
-    frontend_manager
+    frontend_manager::static
```

Comment:
why?

Reply:
inference engine linked statically, tests can be linked statically too, it is better in my opinion